package.json

```diff
@@ -3,25 +3,23 @@
   "description": "Tools for unit testing Lightning web components in a Salesforce DX workspace",
   "version": "0.1.1",
   "dependencies": {
-    "@oclif/command": "^1.8.0",
-    "@oclif/config": "^1.17.0",
-    "@oclif/errors": "^1.3.5",
-    "@salesforce/command": "^3.0.0",
-    "@salesforce/core": "^2.25.1",
+    "@oclif/config": "^1.18.1",
+    "@salesforce/command": "^4.2.0",
+    "@salesforce/core": "^2.31.0",
```

Comment:
These version bumps are the reason for all the changes.  It's necessary so that vulnerabilities showing up in some reports (e.g., npm audit, snyk, etc.) are addressed/removed.